docs_src/general/ServicePorts.md

```diff
@@ -47,11 +47,12 @@ The following tables (organized by type of service) capture the default service
 === "Security"
     |Services Name|	Port Definition|
     |---|---|
-    |kong-db|5432|
-    |vault	|8200|
-    |kong	|8000, 8100, 8443|
+    |kong-db (deprecated)           |5432|
```

Comment:
Deprecated indicates that it is still available, but to recommend to use.
Mark as removed or just remove from the table. 
```suggestion
    |kong-db (removed)           |5432|
```

Reply:
Removed altogether.